### PR TITLE
GH-1881 Default Form Filters

### PR DESCRIPTION
**Pull request by GordonSmith**
Use default values to prime the dashboard.

Fixes GH-1881

Signed-off-by: Gordon Smith <gordonjsmith@gmail.com>

### CODE REVIEW

**Comment by GordonSmith** [comment]
@jbrundage @mlzummo @jchambers-ln Please Review

**Comment by bwood1** [comment]
Looks good.

File: src/marshaller/HipieDDL.js [hunk 1]
```diff
@@ -1060,6 +1069,15 @@
         return this;
     };
 
+    Visualization.prototype.calcRequestFor = function (visualization) {
+        var retVal = {};
+        this.getUpdatesForVisualization(visualization).forEach(function (updatesObj) {
+            //  TODO:  When we support more than "click" this will need enhancment...
```
Comment (jchambers-ln) [inline src/marshaller/HipieDDL.js]: todo?

**Comment by jchambers-ln** [comment]
@GordonSmith just a todo, otherwise looks good

File: src/marshaller/HipieDDL.js (hunk 1 above)
Comment (GordonSmith) [inline src/marshaller/HipieDDL.js]: Yes and real one - when we support more than "click" that loop will need to decide which one to use.

**Pull request merged**